ansible-ipi-install/roles/bootstrap/tasks/20_reprovision_nodes.yml

```diff
@@ -22,40 +22,73 @@
   set_fact:
     os_install: "{{ (rhel_version is version('8.1', '<') or rebuild_provisioner) | ternary('RHEL 8.1', false) }}"
 
+- name: Determine if containers should be used
+  set_fact:
+    run_in_container: "{{ lookup('env', 'DISABLE_PODMAN') != 'true' }}"
```

Comment:
The reference to `DISABLE_PODMAN`  here seems to be counter-intuitive. I understand that this disables running hammercli etc inside a separate container, but this also makes it look like we are not running containerized jetski in a container. Can we come up with something else?